Source/WebInspectorUI/UserInterface/Views/TimelineRecordingContentView.css

```diff
@@ -23,28 +23,30 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+.content-view.timeline-recording {
+    display: flex;
+    flex-direction: column;
+    overflow: auto;
+
+    --timeline-content-browser-min-height: 250px;
```

Comment:
where does this number come from?

Reply:
Arbitrary height to reserve for the content view to be able to represent _some_ meaningful content on very short viewports. Open to other suggestions.